FrontEnd/README.md

```diff
@@ -0,0 +1,7 @@
+### Rules:
+1. Create a folder with a name of your task.
```

Comment:
Take your point, but it seems better to use folder naming appropriate to the name of the page (or component if needed)

Reply:
Fixed as recommended